content/departments/marketing/content/creating_blog_posts.md

```diff
@@ -50,7 +53,7 @@ Your markdown content goes here
 The data between the `---` is called front matter and is used to provide post metadata. Important to note about this metadata, is that:
 
 - The `description` field is optional and only needed if you want to craft the description for your post on the blog the index page.
```

Comment:
I think this should be required since it's used as an excerpt for blog posts on the index.

```suggestion
The `description` field is used as an excerpt for your post on the blog the index page.
```